doc/releases/migration-guide-4.0.rst

```diff
@@ -162,6 +162,68 @@ Bluetooth Classic
 Bluetooth Host
 ==============
 
+ * This release deprecates automatic advertiser resumption.
+
+   This deprecation is compiler-checked. If you get no warnings,
+   you should not be affected.
+
+   Deprecated symbols:
+    * :code:`BT_LE_ADV_OPT_CONNECTABLE`
+    * :code:`BT_LE_ADV_OPT_ONE_TIME`
+    * :code:`BT_LE_ADV_CONN`
+
+   New symbols:
+    * :code:`BT_LE_ADV_OPT_CONN`
+    * :code:`BT_LE_ADV_CONN_FAST_1`
+    * :code:`BT_LE_ADV_CONN_FAST_2`
+
+   `BT_LE_ADV_OPT_CONNECTABLE` is a combined instruction to make the
+   advertiser connectable and to enable automatic resumption. To disable
+   the automatic resumption, use `BT_LE_ADV_OPT_CONN`.
+
+   Extended Advertising API with shorthands
+   =====================================
+
+   Extended Advertising API `bt_le_ext_adv_*` implicitly assume
+   `BT_LE_ADV_OPT_ONE_TIME` and never automatically resume advertising.
```

Comment:
```suggestion
   Extended Advertising API :code:`bt_le_ext_adv_*` implicitly assume
   :code:`BT_LE_ADV_OPT_ONE_TIME` and never automatically resume advertising.
```

Reply:
Pretty sure you either have to do 
```
:code:`bt_whatever`
```
or

```
``bt_whatever``
```

to properly mark things as code

Reply:
This also applies for several other places further below

Reply:
https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#content-highlighting

Guidelines say <code>``</code>.

Reply:
> two backquotes: ``text`` for inline code samples.

Does "inline code samples" imply references for specific values like above? 

Ping @kartben

Reply:
Latest push uses correct [C domain references](https://www.sphinx-doc.org/en/master/usage/domains/c.html#cross-referencing-c-constructs).